GCMessengerSDKSample/GCMessengerSDKSample/ChatWrapperViewController.swift

```diff
@@ -84,6 +83,21 @@ extension ChatWrapperViewController: ChatControllerDelegate {
         case .started:
             print("started")
             stopSpinner(activityView: chatViewControllerActivityView)
+        case .disconnected:
+            
+//            let alert = UIAlertController(title: "Chat was disconnected", message: "We were not able to restore chat connection.\nMake sure your device is connected.\nWould you like to continue with the chat or dismiss it?", preferredStyle: .alert)
+
+            let alert = UIAlertController(title: "Chat was disconnected", message: "We were not able to restore chat connection.\nMake sure your device is connected.", preferredStyle: .alert)
+            
+//            alert.addAction(UIAlertAction(title: "Continue", style: .default, handler: { _ in
+////                self.chatController.continueChat()
+//            }))
+            
+            alert.addAction(UIAlertAction(title: "Dismiss Chat", style: .cancel, handler: { _ in
+                self.dismissChat(nil)
+            }))
+            chatVC.present(alert, animated: true)
```

Comment:
don't you need to add a check to verify that the `chatVC` is not `nil` and still valid (in case we get the event after the chat UI was closed)